src/Foreign/JavaScript/Types.hs

```diff
@@ -64,17 +64,49 @@ This is a record type which has the following fields:
     The initial 'CallBufferMode' to use for 'runFunction'.
     It can be changed at any time with 'setCallBufferMode'.
 
+* @jsSSLBind :: Maybe ByteString@
+
+    Bind address for @HTTPS@.
+    @Nothing@ means that the bind address is read from the environment variable @ADDR@.
```

Comment:
```suggestion
```

All of these fields would become non-nullable and be documented at the `ConfigSSL` type.

Adding a top-level `defaultConfigSSL` which also documents its default values would be handy.